imap/jmap_api.c

```diff
@@ -1228,6 +1235,15 @@ HIDDEN int jmap_findblob(jmap_req_t *req, const char *from_accountid,
 
     syslog(LOG_DEBUG, "jmap_findblob (%s, %s)", from_accountid, blobid);
 
+    if (blob && req->inmemory_blobs) {
+        const struct buf *inmem = hash_lookup(blobid, req->inmemory_blobs);
+        if (inmem) {
+            buf_init_ro(blob, buf_base(inmem), buf_len(inmem));
+            r = 0;
+            goto done;
+        }
+    }
+
```

Comment:
Do I remember correctly from discussion the other day that the point of "findblob_exact" is for limiting the results to things that can be hardlinked?  If so, do we need to exclude in-memory blobs when exact is true?

Reply:
Yes, we do want to exclude if `exact`